src/problem-analysis/target-group-analysis.tex

```diff
@@ -7,11 +7,13 @@ \section{Target group analysis}\label{sec:target-group-analysis}
 
 A good way to do just that is by splitting the target group into different categories.
 It is possible to split the people that play chess by age, skill level, and how frequently they play.
-As the project is about the learning process of chess, dividing the target group by skill level is the most relevant.
-In that case, our target group is beginners.
+As the project is about the learning process of chess, the most relevant choice is to divide the target group based on
+skill level.
+To measure their skill level, the team uses the ELO rating system, which is a method for calculating the relative skill
+levels of players in two-player games such as chess, with the data being taken from \url{Chess.com}.
```

Comment:
"being taken" sound a little agressive IMO. Maybe consider changing to "with data from \url{Chess.com}."